src/collections/_documentation/platforms/python/azure_functions.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Azure Functions
+sidebar_order: 9
+---
+
+Create a .venv in the local machine on VS Code, create a Python function, and install the sentry SDK:
+```python
+sentry_sdk==0.14.4
```

Comment:
Please copy this from the sourcecode of other pages, hardcoding the version here is not good.